failover support

zombiedk opened the pull request:
added config and helptext for setting up failover on the client pc
this is usefull for laptops that need to be able to get updates while they are on a remote network where they cant reach the proxy

zombiedk committed:
added .gitignore

zombiedk committed:
added failover support

zombiedk committed:
better description

zombiedk committed:
moved config to sub dir

zombiedk committed:
sample files

zombiedk committed:
changed to use ENV vars

zombiedk committed:
bug: removed echo

zombiedk committed:
better name

zombiedk committed:
chmod +x

zombiedk committed:
added helptext for sample code

zombiedk committed:
moved back

not part of docker config

zombiedk committed:
changed from ENV to ARG

zombiedk committed:
changed to build time config

zombiedk committed:
define args in .env

zombiedk committed:
added .env file

sameersbn commented:
thanks for the contribution. 
could we move 01proxy and apt-proxy-detect.sh into a sub-directory?

sameersbn commented:
Yea, folder name host is good!

On Thu, Oct 26, 2023, 8:29 PM Mark Petersen ***@***.***>
wrote:

> yes, should I put them in "scripts" or create a folder "host" for them
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sameersbn/docker-apt-cacher-ng/pull/65#issuecomment-1781303513>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADEEI7AJYREMLNT6PBEYVLYBJ3GZAVCNFSM6AAAAAA6FOJUH2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBRGMYDGNJRGM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

zombiedk committed:
moved to host folder

zombiedk commented:
have moved files to host folder

zombiedk committed:
added jet brains

zombiedk committed:
Merge branch 'sameersbn:master' into master

Glorforidor committed:
Use Docker volume

dependabot[bot] committed:
build(deps): bump ubuntu from jammy-20230816 to jammy-20240808

Bumps ubuntu from jammy-20230816 to jammy-20240808.

---
updated-dependencies:
- dependency-name: ubuntu
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump github/codeql-action from 2 to 3

Bumps [github/codeql-action](https://github.com/github/codeql-action) from 2 to 3.
- [Release notes](https://github.com/github/codeql-action/releases)
- [Changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/github/codeql-action/compare/v2...v3)

---
updated-dependencies:
- dependency-name: github/codeql-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump docker/metadata-action from 4 to 5

Bumps [docker/metadata-action](https://github.com/docker/metadata-action) from 4 to 5.
- [Release notes](https://github.com/docker/metadata-action/releases)
- [Upgrade guide](https://github.com/docker/metadata-action/blob/master/UPGRADE.md)
- [Commits](https://github.com/docker/metadata-action/compare/v4...v5)

---
updated-dependencies:
- dependency-name: docker/metadata-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump docker/setup-buildx-action from 2 to 3

Bumps [docker/setup-buildx-action](https://github.com/docker/setup-buildx-action) from 2 to 3.
- [Release notes](https://github.com/docker/setup-buildx-action/releases)
- [Commits](https://github.com/docker/setup-buildx-action/compare/v2...v3)

---
updated-dependencies:
- dependency-name: docker/setup-buildx-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump docker/build-push-action from 4 to 6

Bumps [docker/build-push-action](https://github.com/docker/build-push-action) from 4 to 6.
- [Release notes](https://github.com/docker/build-push-action/releases)
- [Commits](https://github.com/docker/build-push-action/compare/v4...v6)

---
updated-dependencies:
- dependency-name: docker/build-push-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump actions/checkout from 3 to 4

Bumps [actions/checkout](https://github.com/actions/checkout) from 3 to 4.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

zombiedk committed:
Merge pull request #1 from SDFIdk/dependabot/docker/ubuntu-jammy-20240808

build(deps): bump ubuntu from jammy-20230816 to jammy-20240808

zombiedk committed:
Merge pull request #2 from SDFIdk/dependabot/github_actions/github/codeql-action-3

build(deps): bump github/codeql-action from 2 to 3

zombiedk committed:
Merge pull request #3 from SDFIdk/dependabot/github_actions/docker/metadata-action-5

build(deps): bump docker/metadata-action from 4 to 5

zombiedk committed:
Merge pull request #4 from SDFIdk/dependabot/github_actions/docker/setup-buildx-action-3

build(deps): bump docker/setup-buildx-action from 2 to 3

zombiedk committed:
Merge pull request #5 from SDFIdk/dependabot/github_actions/docker/build-push-action-6

build(deps): bump docker/build-push-action from 4 to 6

zombiedk committed:
Merge pull request #6 from SDFIdk/dependabot/github_actions/actions/checkout-4

build(deps): bump actions/checkout from 3 to 4

dependabot[bot] committed:
build(deps): bump docker/login-action from 2 to 3

Bumps [docker/login-action](https://github.com/docker/login-action) from 2 to 3.
- [Release notes](https://github.com/docker/login-action/releases)
- [Commits](https://github.com/docker/login-action/compare/v2...v3)

---
updated-dependencies:
- dependency-name: docker/login-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

zombiedk committed:
Merge pull request #7 from SDFIdk/dependabot/github_actions/docker/login-action-3

build(deps): bump docker/login-action from 2 to 3

dependabot[bot] committed:
build(deps): bump ubuntu from jammy-20240808 to jammy-20250404

Bumps ubuntu from jammy-20240808 to jammy-20250404.

---
updated-dependencies:
- dependency-name: ubuntu
  dependency-version: jammy-20250404
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

JesperKock committed:
Merge pull request #9 from SDFIdk/dependabot/docker/ubuntu-jammy-20250404

build(deps): bump ubuntu from jammy-20240808 to jammy-20250404

JesperKock committed:
Merge pull request #10 from sameersbn/master

build(deps): bump github/codeql-action from 2 to 3 (#68)
